client/post-editor/editor-slug/index.jsx

```diff
@@ -83,6 +85,14 @@ const PostEditorSlug = React.createClass( {
 		}
```

Comment:
do you think it would be possible to update this file to use the newer and preferred referencing for the components here? Instead of using `findDOMNode()`, we can save the reference on instantiation…

```js
<FormTextInput ref={ r => this.slugField = r } … />

focusSlug() {
	if ( this.slugField && this.props.isEditable ) {
		this.slugField.focus();
	}
}
```